frontend/ui/uimanager.lua

```diff
@@ -80,12 +82,11 @@ function UIManager:init()
         end)
     end
     if Device:isPocketBook() then
+        -- Only fg/bg state plugin notifiers, not real power event.
         self.event_handlers["Suspend"] = function()
             self:_beforeSuspend()
-            Device:onPowerEvent("Power")
         end
         self.event_handlers["Resume"] = function()
-            Device:onPowerEvent("Power")
```

Comment:
Note that this doesn't really belong this PR thematically, but I'm keeping it in track with changes to UIManager. See https://github.com/koreader/koreader/pull/6559#discussion_r476898817